components/MainDonationForm.js

```diff
@@ -19,11 +20,35 @@ const AMOUNTS = [
   { value: 25000, label: '$25,000' },
 ];
 
+const CAUSES = [
+  { id: 1, label: 'Students in Need' },
```

Comment:
```suggestion
  { value: 1, label: 'Students in Need' },
```
The API for `SelectFormControl` looks for `value`, not `id`.